Doc/library/unittest.mock.rst

```diff
@@ -609,9 +609,10 @@ the *new_callable* argument to :func:`patch`.
 
         This is either ``None`` (if the mock hasn't been called), or the
         arguments that the mock was last called with. This will be in the
-        form of a tuple: the first member is any ordered arguments the mock
-        was called with (or an empty tuple) and the second member is any
-        keyword arguments (or an empty dictionary).
+        form of a tuple: the first member is ``args`` which will be any ordered
```

Comment:
I'd prefer this to be "the first member, which can also be accessed through the ``args`` property", and likewise with kwargs below.